Gruntfile.js

```diff
@@ -546,12 +574,14 @@ module.exports = function (grunt) {
 
 		if (!grunt.option('no-tests')) {
 			grunt.task.run(['releasetest']);
-			//delete any screenshots that may have happened if it got this far. This isn't foolproof because it relies on the phantomjs server/page timeout, which can take longer than this grunt task depending on how long saucelabs takes to run...
+			// Delete any screenshots that may have happened if it got this far. This isn't foolproof 
```

Comment:
I couldn't read it. It went off the page. I don't have line wrap on, do you? I thought we typically tried to keep code to 80 characters or something.